src/Sarif.Sarifer/source.extension.vsixmanifest

```diff
@@ -24,6 +24,6 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
     <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
   </Assets>
   <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,17.0)" DisplayName="Visual Studio core editor" />
```

Comment:
>17 [](start = 82, length = 2)

Without this change, the VSIX won't install on a machine where VS 2017 is not installed.